tests/exchange/test_exchange.py

```diff
@@ -1168,6 +1200,8 @@ def test_create_order(default_conf, mocker, side, ordertype, rate, marketprice,
 
     assert exchange._set_leverage.call_count == 1
     assert exchange.set_margin_mode.call_count == 1
+    # assert api_mock.create_order.call_args[0][3] == 100
```

Comment:
I couldn't get this working seeing as `amount_to_precision` is mocked. Do you know how to get this working?

Reply:
This would be the only thing that would be left to do on this PR

Reply:
That PR mocks `amount_to_precision` - which handles the conversion, so it's never happening in the first place.

i'm not 100% sure if we do need to test for this explicitly - having proper tests for `amount_to_precision` in place - as well as testing that `create_order` calls `amount_to_precisoin` might suffice.

Reply:
> i'm not 100% sure if we do need to test for this explicitly 

That's enough for me not to do the work, if you ever change your mind I can look at it again